guizlet/frontend-interview
# Compiling example assets with Parcel

- [x] QA @karoun

------------

### What changed?

- Switched from `create-react-app` to [Parcel](https://parceljs.org/) for bundling assets
  - A recent candidate had problems with the former when using NPM
  - The latter supports HMR, many asset types, and languages like TypeScript out of the box
- Added `package-lock.json` files to both examples for NPM compatibility
- Deleted unnecessary files left behind by `create-react-app`
- Updated the various READMEs

------------

### Test plan

- Ensure `yarn && yarn start` successfully runs the dev server in `vanilla-js/`
- Ensure `yarn && yarn start` successfully runs the dev server in `react-js/`
- Ensure `npm install && npm start` successfully runs the dev server in `vanilla-js/`
- Ensure `npm install && npm start` successfully runs the dev server in `react-js/`
